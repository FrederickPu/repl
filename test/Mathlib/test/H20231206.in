{"cmd": "import Mathlib\nimport Mathlib.Tactic.GoalNormalization.NormalizeHyps"}

{"cmd": "theorem problem\n    (b :  ZMod (11^2))\n    (h₀ : b = 24⁻¹) :\n    ∃ (k : ZMod (11^2)), b = k := by\n  normalize_hyps\n  extract_goal\n  sorry",
  "env": 0 }
