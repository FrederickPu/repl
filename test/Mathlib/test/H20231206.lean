import Mathlib
import Mathlib.Tactic.GoalNormalization.NormalizeHyps

/--
info: theorem extracted_1 (h0 : ZMod (11 ^ 2)) (h1 : h0 = 24⁻¹) : ∃ k, h0 = k := sorry
---
warning: declaration uses 'sorry'
-/
#guard_msgs in
theorem problem
    (b :  ZMod (11^2))
    (h₀ : b = 24⁻¹) :
    ∃ (k : ZMod (11^2)), b = k := by
  normalize_hyps
  extract_goal
  sorry
