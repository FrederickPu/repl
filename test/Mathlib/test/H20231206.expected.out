{"env": 0}

{"sorries":
 [{"proofState": 0,
   "pos": {"line": 7, "column": 2},
   "goal": "h0 : ZMod (11 ^ 2)\nh1 : h0 = 24⁻¹\n⊢ ∃ k, h0 = k",
   "endPos": {"line": 7, "column": 7}}],
 "messages":
 [{"severity": "info",
   "pos": {"line": 6, "column": 2},
   "endPos": {"line": 6, "column": 14},
   "data":
   "theorem extracted_1 (h0 : ZMod (11 ^ 2)) (h1 : h0 = 24⁻¹) : ∃ k, h0 = k := sorry"},
  {"severity": "warning",
   "pos": {"line": 1, "column": 8},
   "endPos": {"line": 1, "column": 15},
   "data": "declaration uses 'sorry'"}],
 "env": 1}

